src/search/controllers/search.controller.ts

```diff
@@ -61,6 +65,9 @@ export class SearchController {
     );
   }
 
+  /**
+   * @todo 삭제 예정
+   */
```

Comment:
```suggestion
  /**
   * @deprecated 삭제 예정
   */
```
일반적으로 삭제 예정은 이 태그를 사용합니다.(라이브러리같은거 사용할 때 한번씩 보이는 취소선 생김)
추가적으로 api가 삭제 예정인건 클라이언트 사이드에서도 알아야 할 사항이기에 swagger에도 deprecated 표시가 있으니 추가되면 좋을거같네요
https://jsdoc.app/tags-deprecated